src/public_data/publishing_datasets.md

```diff
@@ -0,0 +1,51 @@
+# Making Datasets Publicly Available
```

Comment:
This is just relevant to internal users. I thought splitting this up and putting it separately is a good idea, since this is not relevant to the general public.

Reply:
Since this is for a different audience and has the feel of a cookbook, it might make sense to have it live under operational cookbooks.